templates/admin/repo/list.tmpl

```diff
@@ -55,8 +55,8 @@
 	</div>
 	<div class="content">
 		<p>{{.i18n.Tr "repo.settings.delete_desc"}}</p>
-		{{.i18n.Tr "repo.settings.delete_notices_2"}}<br>
-		{{.i18n.Tr "repo.settings.delete_notices_fork_1"}}<br>
+		<p>{{.i18n.Tr "repo.settings.delete_notices_2"}}</p>
```

Comment:
I think that the right solution is to simply pass a format argument to `.i18n.Tr "repo.settings.delete_notices_2"` (see [user/dashboard/navbar.tmpl](https://github.com/go-gitea/gitea/blob/f5155b99136b6e19ab494a97adfcee1810a3d5e7/templates/user/dashboard/navbar.tmpl#L45) for an example of what I have in mind)

This would be a one-line change, and would not require new routes or any client-side changes.

Reply:
yes, i think so. but after check in the `repos.go` is imposible, cause the function `deleterepo` will be call after approve button (yes). so, it is imposibble. the issue need name repository when it is deleted.

Reply:
You can add `data-repo-name` to link just like `data-id` is added and use $(this).data('repo-name') in on click javascript to get repo name

Reply:
okay, i will update and update the code